diff -ru --text mongodb-master/SConstruct mongodb-master-new/SConstruct
--- mongodb-master/SConstruct	2019-11-28 01:55:55.000000000 +0000
+++ mongodb-master-new/SConstruct	2021-02-28 02:05:11.028993938 +0000
@@ -1958,36 +1958,6 @@
 
     conf.Finish()
 
-    # If we are using libstdc++, check to see if we are using a libstdc++ that is older than
-    # our GCC minimum of 4.8.2. This is primarly to help people using clang on OS X but
-    # forgetting to use --libc++ (or set the target OS X version high enough to get it as the
-    # default). We would, ideally, check the __GLIBCXX__ version, but for various reasons this
-    # is not workable. Instead, we switch on the fact that _GLIBCXX_PROFILE_UNORDERED wasn't
-    # introduced until libstdc++ 4.8.2. Yes, this is a terrible hack.
-    if usingLibStdCxx:
-        def CheckModernLibStdCxx(context):
-            test_body = """
-            #define _GLIBCXX_PROFILE
-            #include <unordered_map>
-            #if !defined(_GLIBCXX_PROFILE_UNORDERED)
-            #error libstdc++ older than 4.8.2
-            #endif
-            """
-
-            context.Message('Checking for libstdc++ 4.8.2 or better... ')
-            ret = context.TryCompile(textwrap.dedent(test_body), ".cpp")
-            context.Result(ret)
-            return ret
-
-        conf = Configure(myenv, help=False, custom_tests = {
-            'CheckModernLibStdCxx' : CheckModernLibStdCxx,
-        })
-
-        if not conf.CheckModernLibStdCxx():
-            myenv.ConfError("When using libstdc++, MongoDB requires libstdc++ 4.8.2 or newer")
-
-        conf.Finish()
-
     if has_option("use-glibcxx-debug"):
         # If we are using a modern libstdc++ and this is a debug build and we control all C++
         # dependencies, then turn on the debugging features in libstdc++.
